crates/biome_js_analyze/src/analyzers/nursery/no_implicit_any_let.rs

```diff
@@ -0,0 +1,93 @@
+use biome_analyze::{context::RuleContext, declare_rule, Ast, Rule, RuleDiagnostic};
+use biome_console::markup;
+use biome_js_syntax::{JsFileSource, JsVariableDeclaration, JsVariableDeclarator};
+
+declare_rule! {
+    /// Disallow use of implicit `any` type on variable declarations.
+    ///
+    /// Typescript variable declaration without any `type` or `initialization` can cause issue later in the code.
```

Comment:
nits:
```suggestion
    /// TypeScript variable declaration without any `type` or `initialization` can cause issue later in the code.
```

After fixing this, you need to run `just codegen` again 🙏

Reply:
And this @Conaclos 's suggest is good to add.
https://github.com/biomejs/biome/pull/395#discussion_r1334961574

Reply:
I've fixed it!